subsys/bluetooth/mesh/access.c

```diff
@@ -465,15 +466,71 @@ struct bt_mesh_elem *bt_mesh_elem_find(uint16_t addr)
 		}
 	}
 
+	return NULL;
+}
+
+bool bt_mesh_has_addr(uint16_t addr)
+{
+	uint16_t index;
+
+	if (BT_MESH_ADDR_IS_UNICAST(addr)) {
+		return bt_mesh_elem_find(addr) != NULL;
+	}
+
+	if (msg_cb) {
+		return true;
+	}
+
 	for (index = 0; index < dev_comp->elem_count; index++) {
 		struct bt_mesh_elem *elem = &dev_comp->elem[index];
 
 		if (bt_mesh_elem_find_group(elem, addr)) {
-			return elem;
+			return true;
 		}
 	}
 
-	return NULL;
+	return false;
+}
+
+void bt_mesh_msg_cb_set(void (*cb)(uint32_t opcode, struct bt_mesh_msg_ctx *ctx,
+			struct net_buf_simple *buf))
+{
+	msg_cb = cb;
+}
+
+int bt_mesh_msg_send(struct bt_mesh_msg_ctx *ctx, struct net_buf_simple *buf)
```

Comment:
Yes, I agree that this is really more of a wrapper function currently. My goal was create a nice clean function that was visible to the application. So, your suggestion to make bt_mesh_trans_send more friendly works for me. In this case, what would be the process for making bt_mesh_trans_send visible to the application for use? 

You're probably right that these changes to bt_mesh_trans_send are out of the scope of this PR. I on board with creating a separate PR for this.

Reply:
> what would be the process for making bt_mesh_trans_send visible to the application for use?

It would not be visible to the application by default, unless the app explicitly adds subsys/bluetooth/mesh to its list of include directories. My assumption is that a module that does this could be added as a zephyr library, as it would be touching internal stuff. This can be done from any directory, using this pattern in cmake:
```
zephyr_library()
zephyr_library_link_libraries(subsys__bluetooth)
zephyr_library_sources(my_file.c)
```
Now, all subsys/bluetooth/mesh headers can be included from my_file.c as `#include "mesh/transport.h"`